sydent/users/tokens.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import time
+
+from sydent.util.tokenutils import generateAlphanumericTokenOfLength
+from sydent.db.accounts import AccountStore
+
+logger = logging.getLogger(__name__)
+
+def issueToken(sydent, user_id):
```

Comment:
Any reason this is not part of the account store?

Reply:
Just that the actual issuing of a token seemed beyond the remit of the store, ie. the store should just be doing the storing.